torch/_dynamo/variables/tensor.py

```diff
@@ -241,14 +241,14 @@ def create(cls, tx, proxy, example_value=None, **options):
             return TorchVariable(proxy.node.target)
         elif istype(example_value, (int, bool, float)) and config.dynamic_shapes:
```

Comment:
Does this check need to be `istype(example_value, (int, bool, float, torch.SymIntNode, torch.SymFloatNode))`?

Not an expert on this logic, but it looks like `example_value` could come from an output of a previous graph - which could be a symint?

Reply:
nvm ignore - just saw the `SymInt` handling further down